platforms/core-runtime/files/src/main/java/org/gradle/internal/file/nio/PosixFilePermissionConverter.java

```diff
@@ -14,14 +14,23 @@
  * limitations under the License.
  */
 
-package org.gradle.internal.nativeintegration.filesystem.jdk7;
+package org.gradle.internal.file.nio;
 
 import java.nio.file.attribute.PosixFilePermission;
 import java.util.EnumSet;
 import java.util.Set;
 
-import static java.nio.file.attribute.PosixFilePermission.*;
+import static java.nio.file.attribute.PosixFilePermission.GROUP_EXECUTE;
+import static java.nio.file.attribute.PosixFilePermission.GROUP_READ;
+import static java.nio.file.attribute.PosixFilePermission.GROUP_WRITE;
+import static java.nio.file.attribute.PosixFilePermission.OTHERS_EXECUTE;
+import static java.nio.file.attribute.PosixFilePermission.OTHERS_READ;
+import static java.nio.file.attribute.PosixFilePermission.OTHERS_WRITE;
+import static java.nio.file.attribute.PosixFilePermission.OWNER_EXECUTE;
+import static java.nio.file.attribute.PosixFilePermission.OWNER_READ;
+import static java.nio.file.attribute.PosixFilePermission.OWNER_WRITE;
 
+@SuppressWarnings({"Since15", "OctalInteger"})
```

Comment:
❓ Also here, is it safe to suppress "Since15"? 

Should we add a comment why it is safe?